docker/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM python:3.9-slim
+
+COPY . /archive-db
+WORKDIR /archive-db
+
+RUN \
+  apt-get update && \
+  apt-get install -y git && \
+  sed -i -re 's/(port:).*/\1 8787/' config/app.config && \
+  python3 -m venv --upgrade-deps .venv && \
```

```suggestion
  rm -rf .venv && \
  python3 -m venv --upgrade-deps .venv && \
```
I think I found an interesting issue on this one (but it's actually valid for all three PRs). If there is already a `.venv` file, it will be copied in the docker container and will interfere with the new virtual environment there.